src/SqlEditor.css

```diff
@@ -0,0 +1,20 @@
+.SqlEditor{
+    box-sizing: border-box;
+    margin: auto;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+}
+
+.SqlEditor textarea{
+    height: 5em;
+    resize: none;
+    border-style: solid;
+    background-color: lightgray;
+
+}
+
+.error{
```

Comment:
This should be `SqlEditor.error` (you can set `className` in the error case appropriately so this applies).